tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/Settings.cs

```diff
@@ -10,6 +10,7 @@ public Settings(IConfiguration config)
             ConnectionString = config.GetConnectionString("OrderingDb");
             OrderingApiBaseUrl = config.GetValue<string>("OrderingApiBaseUrl");
             Authority = config.GetValue<string>("Authority");
+            TestTimeOutInSeconds = config.GetValue<int>("TestTimeOutInSeconds");
```

Comment:
Could the name of this be improved. At first glance I am not sure what time out it is for. How about 'OrderingApiHealthCheckTimeout'?